my_cinema/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for my_cinema project.
+
+Generated by "django-admin startproject" using Django 5.1.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / "subdir".
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-ga67d=g+(@sbbea0fp+y&tw-_6sto%g4tr^mohs-aqj94%y(+7"
```

Comment:
The `SECRET_KEY` is hardcoded and exposed in the settings file. It's recommended to load this key from an environment variable or a separate configuration file for security reasons.